src/illuminatio/illuminatio.py

```diff
@@ -1,33 +1,40 @@
+"""
+This file contains the illuminatio CLI
+"""
+
 import logging
 import time
 from os import path
+import json
 
 import click
 import click_log
-import json
 import kubernetes as k8s
 import yaml
 from illuminatio.cleaner import Cleaner
 from illuminatio.test_generator import NetworkTestCaseGenerator
 from illuminatio.test_orchestrator import NetworkTestOrchestrator
 from illuminatio.util import CLEANUP_ALWAYS, CLEANUP_ON_REQUEST
 
-logger = logging.getLogger(__name__)
-click_log.basic_config(logger)
+LOGGER = logging.getLogger(__name__)
+click_log.basic_config(LOGGER)
 
 # ToDo do we really need here global variables?
-orch = None
-gen = None
+ORCH = None
+GENERATOR = None
 
 
 @click.group(chain=True)
-@click_log.simple_verbosity_option(logger, default="INFO")
+@click_log.simple_verbosity_option(LOGGER, default="INFO")
 @click.option('--incluster', default=False, is_flag=True)
 def cli(incluster):
-    global orch
-    orch = NetworkTestOrchestrator([], logger)
-    global gen
-    gen = NetworkTestCaseGenerator(logger)
+    """
+    initializes global variables
+    """
+    global ORCH
+    ORCH = NetworkTestOrchestrator([], LOGGER)
+    global GENERATOR
+    GENERATOR = NetworkTestCaseGenerator(LOGGER)
```

Comment:
I would prefer moving `ORCH` and `GENERATOR` into the according methods since `GENERATOR` is only used in `run` and the `NetworkTestOrchestrator` can be created in `run` and `clean`, this way we can prevent the global variable (and there is no need for a global variable probably a relict)